tests/run-centos-ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+pip install virtualenv
+pip install python
+virtualenv --system-site-packages env
+source env/bin/activate
+
+# install dependency packages
+yum install libselinux-python
+yum install gcc python-virtualenv
+pip install ansible molecule docker-py
+
+# prerequisites to install docker
+sudo yum install -y yum-utils \
```

Comment:
Can you combine this into the yum install above. You don't need sudo since you'll be running this as root.